src/TransitionSystems/quotient.rs

```diff
@@ -164,7 +164,7 @@ impl TransitionSystem for Quotient {
         if location.is_inconsistent() {
             //Rule 10
             if is_input {
-                let mut transition = Transition::new(location, self.dim);
+                let mut transition = Transition::new(TransitionID::Simple(format!("I|Inconsistent:{}", location.id)), location, self.dim);
```

Comment:
Must this format be different from 
147 compiled components .rs
return vec![Transition::new(TransitionID::Simple(format!(**"Inconsistent:{}"**, locations.id)), locations, self.dim)];